pinned-react/src/pages/Board/EventsPage.js

```diff
@@ -20,33 +20,47 @@ const EventsPage = () => {
      })
    }, [id])
 
+  const PreviewImage = ({ preview }) => {
```

Comment:
extract preview image to it own seperate file in the components folder, because it will be used all across the site. And then import it from there